### ISSUE TITLE
Was working fine but now getting lots of the same error message

### ISSUE THREAD

**Issue by nothing2obvi** [hollanbm/sonarr-series-scanner#20]
```
2024-05-14 02:12:06.153 | ERROR | __main__:start:75 |  |  | ('Schema validation error', SchemaError("Key 'sonarr' error:\nOr({'name': And(<function <lambda> at 0xffff94684b80>, Use(<class 'str'>), <function <lambda> at 0xffff94684180>), 'url': And(<function <lambda> at 0xffff94713e20>, Use(<class 'str'>), <function <lambda> at 0xffff94713ec0>), 'api_key': And(<function <lambda> at 0xffff94713f60>, Use(<class 'str'>), <function <lambda> at 0xffff94720040>), 'series_scanner': {'enabled': <class 'bool'>, 'hourly_job': <class 'bool'>, Optional('hours_before_air'): <class 'int'>}, 'existing_renamer': {'enabled': <class 'bool'>, 'hourly_job': <class 'bool'>}}) did not validate {'name': 'Sonarr', 'url': 'http://10.0.0.94:8989', 'api_key': 'api-key-here', 'series_scanner': {'enabled': True, 'hourly_job': True, 'hours_before_air': 4}}\nMissing key: 'existing_renamer'"))
```

**Comment by hollanbm** [comment]
> ```
> 
> 2024-05-14 02:12:06.153 | ERROR | __main__:start:75 |  |  | ('Schema validation error', SchemaError("Key 'sonarr' error:\nOr({'name': And(<function <lambda> at 0xffff94684b80>, Use(<class 'str'>), <function <lambda> at 0xffff94684180>), 'url': And(<function <lambda> at 0xffff94713e20>, Use(<class 'str'>), <function <lambda> at 0xffff94713ec0>), 'api_key': And(<function <lambda> at 0xffff94713f60>, Use(<class 'str'>), <function <lambda> at 0xffff94720040>), 'series_scanner': {'enabled': <class 'bool'>, 'hourly_job': <class 'bool'>, Optional('hours_before_air'): <class 'int'>}, 'existing_renamer': {'enabled': <class 'bool'>, 'hourly_job': <class 'bool'>}}) did not validate {'name': 'Sonarr', 'url': 'http://10.0.0.94:8989', 'api_key': 'api-key-here', 'series_scanner': {'enabled': True, 'hourly_job': True, 'hours_before_air': 4}}\nMissing key: 'existing_renamer'"))
> 
> ```

I added another job to the app, called `existing_renamer` -- you can read more ab that [here](https://github.com/hollanbm/sonarr-series-scanner?tab=readme-ov-file#existing-renamer)

I've also updated the [config settings](https://github.com/hollanbm/sonarr-series-scanner?tab=readme-ov-file#configuration) in the README.md

As well as the [config.yml.example](https://github.com/hollanbm/sonarr-series-scanner/blob/main/docker/config.yml.example)

I do think this isn't a very good experience though. In the future, I'll find a way of making new configurations optional, or possibly even update the file beforehand. That way you, or anybody else, won't run into this again

TL;DR - add the new `existing_renamer` settings to your `config.yml`

```yaml
sonarr:
  - name: tv
    url: https://sonarr.tld:8989
    api_key: not-a-real-api-key
    series_scanner:
      enabled: True
      hourly_job: True
    existing_renamer:
      enabled: False
      hourly_job: False
```

**Comment by hollanbm** [comment]
@nothing2obvi 

I've published `0.5.1` which makes the configuration settings optional.

You can update your app version to `0.5.1`, or update your `config.yml` or both 💯
